swift/vapor-framework/config.yaml

```diff
@@ -3,5 +3,8 @@ framework:
   version: 4.7
   name: vapor
 
-options: serve --hostname 0.0.0.0 --port 3000 --env prod
+environment:
+  SERVER_HOSTNAME: 0.0.0.0
```

Comment:
Vapor won't automatically pick up these environment variables (yet - there is an open discussion about that, so this would require code changes as well to read these)

Reply:
I thought it was the case since https://github.com/vapor/vapor/issues/2333#issuecomment-624963814

btw, this is working,  but perhaps `0.0.0.0:3000` is the default

Reply:
I thought it was the case since https://github.com/vapor/vapor/issues/2333#issuecomment-624963814

btw, this is working,  but perhaps `0.0.0.0:3000` is the default

Reply:
Yeah it's hardcoded in currently, I can submit a PR to change that if you like?

Reply:
sure, it'll be a pleasure, taking from `env` is easier to maintain :heart:

Reply:
@0xTim any time for this `PR`  ?

Reply:
@waghanza yeah I'll try and get some time to do it tonight otherwise I'll do it tomorrow morning

Reply:
@waghanza couldn't push to your branch so I've created #2919